material.css

```diff
@@ -270,3 +270,24 @@ input.Button[type="submit"] {
     border-radius: 18px;
     margin-left: 6px;
 }
+
+.SessionDropdown .Dropdown-toggle .Avatar {
+    margin: 6px 5px -2px -6px;
+    width: 24px;
+    height: 24px;
+    border-radius: 24px;
+    font-size: 12px;
+    line-height: 24px;
+}
+
+@media (min-width: 768px)
+.NotificationsDropdown .Dropdown-toggle {
+    width: 36px;
+    text-align: center;
+}
+
+@media (min-width: 768px)
+.DiscussionPage-nav .ButtonGroup:not(.itemCount1) .Dropdown-toggle {
```

Comment:
Can you tell me what does this do?

Reply:
Let me review that, one sec.

Reply:
If you go here: https://flightsimtr.tk/d/14-forum-kurallarimiz as a test topic, the dropdown button with arrow. This will fix it colliding with other buttons.